docs/schema-v2.md

```diff
@@ -5,12 +5,413 @@ This specification defines a format for a set of rules for the purpose of custom
 
 **This format will be updated in the future. However, we assure you that it will not break existing skyboxes.**
```

Comment:
Honestly should remove the promise to not break existing skyboxes.  I've already done it a number of times.  (or I figure out how to properly change it so that both versions still work)